src/Server/Resource/DoctrineResource.php

```diff
@@ -2,25 +2,28 @@
 
 namespace ZF\Apigility\Doctrine\Server\Resource;
 
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManagerInterface;
```

Comment:
Other than the interface (which I'll remove) I don't believe anything is ORM specific BUT I haven't used Doctrine ODM at all and it would be prudent to have someone familiar with it review the final PR to make sure the concepts will carry over correctly